### ISSUE TITLE
[react-i18n] Humanized date does not completely match Polaris guidelines

### ISSUE THREAD

**Issue by tigercao1** [Shopify/quilt#2175]
## Overview

TL;DR Using `formatDate` with `DateStyle.Humanize` does not completely follows the [Polaris Date guidelines](https://polaris.shopify.com/content/grammar-and-mechanics#section-dates-numbers-and-addresses). But on `quilt` documentation it states that `DateStyle.Humanize` adheres to the guideline

Time created within the minute should be displayed as `Just now` but using `formatDate` with `DateStyle.Humanize` returns us `Today`

![1](https://user-images.githubusercontent.com/32528094/155580824-ccf71752-6b2b-47b0-853f-eeacb7bc9b1b.png)
![2](https://user-images.githubusercontent.com/32528094/155580837-7dfb1ac7-c09e-42ab-a595-7d1f786549b2.png)

Time created within the hour should be displayed as `x minutes ago` but using `formatDate` with `DateStyle.Humanize` returns us `Today`

![3](https://user-images.githubusercontent.com/32528094/155580874-25974755-a63c-48fb-b95b-551c680e70c7.png)
![4](https://user-images.githubusercontent.com/32528094/155580895-6ce843c9-9d3f-48f0-ae62-dd7654811cc9.png)

Time created within the day displays correctly with the time

![5](https://user-images.githubusercontent.com/32528094/155580948-3dfea76c-8b08-4c93-b823-c0f4579ecf76.png)
![6](https://user-images.githubusercontent.com/32528094/155580964-dbf6552e-e50e-42ac-8a1b-a24da1b1698a.png)

Time created yesterday is displayed as `Yesterday` using `formatDate` with `DateStyle.Humanize` Should be displayed as `Yesterday at xx:xx pm`

![7](https://user-images.githubusercontent.com/32528094/155580976-54da6055-baec-4d74-8116-f35fb2a8e388.png)
![8](https://user-images.githubusercontent.com/32528094/155580982-c19e1ebc-8412-467f-9b65-2440ac757a7b.png)

Time created within the week displays correctly.

![9](https://user-images.githubusercontent.com/32528094/155581003-db608452-b418-4484-a520-51ad3ff8d171.png)
![10](https://user-images.githubusercontent.com/32528094/155581011-dd396f79-ddc6-4041-afd8-33c66d7acc00.png)

Time created within the year displays correctly

![11](https://user-images.githubusercontent.com/32528094/155581030-5976f2c1-e9d6-480a-99bb-f6e5a7ecd1aa.png)
![12](https://user-images.githubusercontent.com/32528094/155581041-38adbfef-b2c0-4615-95ee-39c59a593d81.png)

Time created over a year displays correctly

![13](https://user-images.githubusercontent.com/32528094/155581063-259f11c4-6c09-40f8-bb02-82191df16b8c.png)
![14](https://user-images.githubusercontent.com/32528094/155581071-a6279651-33a2-4de6-b335-73937c5a5edc.png)

## Consuming repo

Web - Customers

---

## Checklist

- [x] Please delete the labels section before submitting your issue
- [x] I have described this issue in a way that is actionable (if possible)

**Comment by tigercao1** [comment]
Created by mistake, closing now

**Comment by tigercao1** [comment]
After some investigation, this is a none issue. To correctly use `i18n-formatDate` with `DateStyle.Humanize` to match Polaris date style guidelines, we need to provide translations in the consuming repo. See the `migration-guide` [here](https://github.com/Shopify/quilt/blob/main/packages/react-i18n/migration-guide.md).

Example use case:
In consuming repo .tsx file:
<img width="444" alt="image" src="https://user-images.githubusercontent.com/32528094/156400097-61475a09-0d19-4b80-b793-6b21888bdb9d.png">

In consuming repo translation file:
<img width="390" alt="image" src="https://user-images.githubusercontent.com/32528094/156400189-ccd53b80-35a0-4acb-ba79-880596650919.png">
